brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerResponseAdapter.java

```diff
@@ -18,12 +18,8 @@ public HttpServerResponseAdapter(HttpResponse response)
 
     @Override
     public Collection<KeyValueAnnotation> responseAnnotations() {
-        int httpStatus = response.getHttpStatusCode();
-
-        if ((httpStatus < 200) || (httpStatus > 299)) {
-            KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create("http.responsecode", String.valueOf(httpStatus));
-            return Arrays.asList(statusAnnotation);
-        }
-        return Collections.EMPTY_LIST;
+        KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create(
```

Comment:
Here's the challenge..

Say someone wants to change a documented policy, like to no longer
conditionally log 200 response, something that we took time to
document, put into thrifts, etc. Why not add something nice during the
process.. like using consistent keys?

Then, when we talk about release notes, we can say what the benefit
was that came along with the break.

It is also nicer to folks who maintain docs and constants (like me).
I'd rather to have some help, vs just cleanup work changing the
thrifts etc to say we don't advocate dropping response code on 2xx
anymore. I feel like even doing a global replace for using the
constant keys is a reasonable ask when there's work being added to my
plate. Do you disagree?